config_defaults_inc.php

```diff
@@ -1955,6 +1955,20 @@
  */
 $g_file_upload_method = DATABASE;
 
+/**
+ * Use Zip_attachments: zip selected file types in order to optimize performance 
+ * and save disk/db space
+ * @global integer $g_zip_attachments
+ */
+$g_zip_attachments		=	ON;
+
+/**
+ * Use Zip_excludes: define the file types to ommit from the zipping process 
+ * because the files are already compressed and/or one does not want to compress a specific file type
+ * @global integer $g_zip_excludes
+ */
+$g_zip_excludes			=	"pdf,zip,jpg,docx,xlsx,pptx,x-zip-compressed";
```

Comment:
Single-quoted strings are preferred.

And it maybe better to define the config as an array of strings, for consistency with $g_preview_text_extensions and others.

Indicate that values should be provided in lower case.